lib/tasks/cucumber.rake

```diff
@@ -32,6 +32,18 @@ begin
       t.profile = 'rerun'
     end
 
+    Cucumber::Rake::Task.new({:aa => 'test:prepare'}, 'Run aa features that should pass') do |t|
```

Comment:
It's the description of the task. if you run `$ rake -T` you can see all tasks with their descriptions.